toolsrc/include/vcpkg/vcpkgcmdarguments.h

```diff
@@ -154,6 +154,10 @@ namespace vcpkg
         constexpr static StringLiteral WAIT_FOR_LOCK_SWITCH = "x-wait-for-lock";
         Optional<bool> wait_for_lock = nullopt;
 
+        constexpr static StringLiteral VCPKG_EXPERIMENTAL_MIRROR_URL = "VCPKG_EXPERIMENTAL_MIRROR_URL";
```

Comment:
We conventionally use lowercase here
```suggestion
        constexpr static StringLiteral VCPKG_EXPERIMENTAL_MIRROR_URL = "x-mirror-url";
```

Reply:
The environment variable name is set here. Is this correct?